commonui/src/main/java/com/hyperwallet/android/ui/common/view/TransferMethodUtils.java

```diff
@@ -181,4 +184,21 @@ private static String getFourDigitsIdentification(@NonNull final Context context
 
         return context.getString(stringResId, identificationText);
     }
+
+    private static String getFourDigitsIdentificationWithCardBrand(@NonNull final Context context,
+            @NonNull final TransferMethod transferMethod,
+            @NonNull @TransferMethod.TransferMethodFieldKey final String cardNumberFieldKey,
+            @NonNull @TransferMethod.TransferMethodFieldKey final String cardBrandFieldKey) {
```

Comment:
`cardNumberFieldKey` and `cardBrandFieldKey` parameter values are from static constant, why is it being passed as a parameter? couldn't we just use the constant value directly without passing it as a parameter?

Reply:
effectively using the following ?
```
transferMethod.getField(CARD_NUMBER);
transferMethod.getField(CARD_BRAND);
```

Reply:
fixed.